### DISCUSSION TITLE
Que el updateproperty, pueda, agregar o reemplazar subproperties o schemasinstance sin pisar lo ya existente

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#812] [3. Ideas & Feature Requests]
Hoal a todos

Levanto esto, porque he tenido problemas cuando quiero actualizar algún valor de una property, y cuya property ya cuenta con valores. SI uso Updateproperty, me pisará los datos, dejando solo el que actualicé.

Por ende, sería bueno que el u0pdate property, pueda reemplazar datos existentes, sin borrar los que no quiero reemplzar. agregar  nuevos datos sin borrar los ya existentes (esto en tema de schemainstance)

En caso que ese es el verdadero comportamiento me avisan para crear el issue.

Salu2

**Comment by acedward** [comment]
Como alternativa puedes usar `PATCH /v2/properties/jsonpatch/:id` permite hacer operaciones de actualización a campos específicos

**Reply by acedward** [reply]
No es un bug propiamente tal, porque no mandarlo es equivalente a decir borralo (e.g., como lo harias para borrarlo?)
Pero entiendo que puede ser confuso.
